src/Components/Pages/LoginPage.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import LoginForm from './LoginForm/LoginForm'
+
+export default function LoginPage() {
+  return (
+    <div className='LoginPage-container'>
```

Comment:
Je dirais plutôt tout en CamelCase
```suggestion
    <div className='LoginPageContainer'>
```